### ISSUE TITLE
the optimizer stays in the background and does not let you see the interface

### ISSUE THREAD

**Issue by LionART123418** [hellzerg/optimizer#283]
Hello, my problem is that my optimizer does not open and when I try to open it again I get that it is happening in the background. I tried to delete the optimizer folder from the program data but I can't find it. Do you have a solution since I enabled some components that are affecting the pc.

It would be of great help if you recommend me what to do.

**Comment by hellzerg** [comment]
There is the Optimizer folder in C:\ProgramData

If you type the location in start menu or location bar in Files Explorer, you will get there.

**Comment by LionART123418** [comment]
Well, I just found the optimizer folder but it doesn't work, it's still running in the background and it doesn't show the interface, it stays in "checking the requirements"

**Comment by hellzerg** [comment]
Did you kill the process, before deleting the folder?

**Comment by LionART123418** [comment]
yes, is there a problem with that or was it ok?

**Comment by hellzerg** [comment]
It's okay, can you run the app again and send me the optimizer.log file here?

**Comment by LionART123418** [comment]
of course, I just don't know where to find said file I'm not that good at this sorry

**Comment by hellzerg** [comment]
It's in the Optimizer folder, inside the C:\ProgramData.

**Comment by LionART123418** [comment]
I entered the folder but I only get the optimizer.json file, the others do not indicate the name that you mention
{
  "Theme": "153, 102, 204",
  "AppsFolder": "C:\\Users\\HP\\Desktop\\Optimizer Downloads",
  "EnableTray": false,
  "AutoStart": false,
  "LanguageCode": 5,
  "EnablePerformanceTweaks": false,
  "DisableNetworkThrottling": false,
  "DisableWindowsDefender": false,
  "DisableSystemRestore": false,
  "DisablePrintService": false,
  "DisableMediaPlayerSharing": false,
  "DisableErrorReporting": false,
  "DisableHomeGroup": false,
  "DisableSuperfetch": false,
  "DisableTelemetryTasks": false,
  "DisableCompatibilityAssistant": false,
  "DisableFaxService": false,
  "DisableSmartScreen": false,
  "DisableCloudClipboard": false,
  "DisableStickyKeys": false,
  "DisableHibernation": false,
  "DisableSMB1": false,
  "DisableSMB2": false,
  "DisableNTFSTimeStamp": false,
  "DisableSearch": false,
  "DisableOffice2016Telemetry": false,
  "DisableVisualStudioTelemetry": false,
  "DisableFirefoxTemeletry": false,
  "DisableChromeTelemetry": false,
  "DisableNVIDIATelemetry": false,
  "DisableOneDrive": false,
  "EnableLegacyVolumeSlider": false,
  "DisableQuickAccessHistory": false,
  "DisableStartMenuAds": false,
  "UninstallOneDrive": false,
  "DisableMyPeople": false,
  "DisableAutomaticUpdates": false,
  "ExcludeDrivers": false,
  "DisableTelemetryServices": false,
  "DisablePrivacyOptions": false,
  "DisableCortana": false,
  "DisableSensorServices": false,
  "DisableWindowsInk": false,
  "DisableSpellingTyping": false,
  "DisableXboxLive": false,
  "DisableGameBar": false,
  "DisableInsiderService": false,
  "DisableStoreUpdates": false,
  "EnableLongPaths": false,
  "RemoveCastToDevice": false,
  "EnableGamingMode": false,
  "TaskbarToLeft": false,
  "DisableSnapAssist": false,
  "DisableWidgets": false,
  "DisableChat": false,
  "TaskbarSmaller": false,
  "ClassicRibbon": false,
  "ClassicMenu": false,
  "DisableTPMCheck": false,
  "CompactMode": false,
  "DisableStickers": false,
  "DisableVBS": false
}
is this the correct one?

**Comment by hellzerg** [comment]
Okay.

Try downloading .NET Framework 4.8 from Microsoft using this link
https://dotnet.microsoft.com/en-us/download/dotnet-framework/thank-you/net48-web-installer

Install it, then restart your PC and try running the app again.

**Comment by LionART123418** [comment]
It says that I already have it installed

**Comment by hellzerg** [comment]
You can't see the loading screen when you run the app?

**Comment by LionART123418** [comment]
Yes, I see it, but as I explained, it only stays on the loading screen saying "checking the requirements" It does not advance and when I try to start it again it tells me that it is in the background

**Comment by hellzerg** [comment]
Do you want to connect to you via TeamViewer so we can debug the problem?

@LionART123418

**Comment by LionART123418** [comment]
Yes, only that I would have to put the PC in English since I am a Spanish-speaking user, or is there no problem with that?

**Comment by hellzerg** [comment]
It would be helpful if you can do it.

**Comment by LionART123418** [comment]
Well, I tried to change the language but I can't due to the version of my windows, which is home, however, the rest, such as the keyboard, are in English, an apology 
well what do i have to do

**Comment by hellzerg** [comment]
Come over to Optimizer Discord server.
